book/website/community-handbook/translation/localisation.md

```diff
@@ -0,0 +1,45 @@
+(ch-translation-localisation)=
+
+## Localisation Platforms and Translation
+
+Localisation involves more than just translation, which only transforms text. Localisation addresses other factors such as text length and cultural references.  **A Translation Management System (TMS)** manages the localisation process from the beginning of a translation process until the finished product. They are widely used in Open source projects because they offer many advantages such as workflow, automation, transparency, and fast project delivery.
+
+```{figure} ../../figures/Translation_management_systems.jpg
+---
+height: 500px
+name: Translation
+alt: This illustration shows the features of Translation Management System. It contains Translation Memory, Machine Translation, glossary, QA checks which are easily integrated with GitHub.
+---
+_The Turing Way_ project illustration by Scriberia.
+Used under a CC-BY 4.0 licence.
+DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807)
+```
+
+### Features of Localisation Platforms:
+
+- **Translation Memory**
+
+Translation Memory can be described as a database of sentences or texts and their translations that can be automatically reused in/outside the project. Translation Memory can be very powerful in ensuring consistent and high-quality translations across different projects.
+- **Glossary**
+
+A glossary is a collection of the key terms that are used in the project to ensure consistency. The glossary helps in creating, storing, and managing all the project terminology in one place.
+- **Machine Translation**
+
+Machine Translation can speed up the translation process. It provides translation suggestions from automatic translation services like Google Translate and AutoML Translation, Microsoft Translate, and more. 
+- **Integration with GitHub**
+
+Continuous integration is vital for automation in open sources projects. Many localisation platforms offer integration with GitHub, which synchronises source and translation files between your GitHub repository and translation project.
+- **Screenshots for the context**
+
+Screenshots provide context to the text for translation. TMS offer the opportunity of attaching screenshots to each of the phrases to provide a visual context to translators
+- **QA checks**
+
+Many TMS support QA checks. These can include spelling and/or grammatical errors, inconsistent placeholders, and unbalanced brackets. These QA checks make the translation process smoother for translators.
+
+
+### Examples for Localisation Platforms:
```

Comment:
I would move this section upwards, just before the features because examples help make the concept less abstract.

Reply:
That's really a good idea! I'll do that - thank you!